pages/platform/kubernetes-k8s/resizing-persistent-volumes/guide.en-gb.md

```diff
@@ -282,7 +282,7 @@ We verify that the volume has been expanded:
 kubectl describe pvc mysql-pv-claim
 ```
 
-We will get a message that the PVC is waiting for user to start a pod to finish file system resize of the volume.
+In the conditions we can see that the PVC is waiting for user to start a pod to finish file system resize of the volume.
```

Comment:
s/In the conditions/In these conditions,/ ??
Or maybe this sentence needs to be rework?

Reply:
I'm talking about the field "Conditions" when you do a describe on the PVC.
Maybe I should add quotes ?

Reply:
Or add something like: 
`In the "conditions" field shown by the output of the previous command line, we can see [...]`
WDYT?